lib/hypernova/plugins/development_mode_plugin.rb

```diff
@@ -1,6 +1,6 @@
 require 'erb'
 
-class DevelopmentModePlugin
+class Hypernova::DevelopmentModePlugin
```

Comment:
We should probably move the file into a hypernova dir to be consistent with ruby conventions.